app_dart/lib/src/model/ci_yaml/target.dart

```diff
@@ -240,6 +240,16 @@ class Target {
     return value.bringup ? 'staging' : 'prod';
   }
 
+  /// Get the runIf globs for when this target should run.
+  ///
+  /// If not empty, will include default globs that should trigger all tests.
+  List<String> get runIf {
+    if (value.runIf.isEmpty) {
+      return <String>[];
+    }
+    return <String>['.ci.yaml', 'DEPS', ...value.runIf];
```

Comment:
Is this only for simplicity in .ci.yaml? It seems more clear to me to have `.ci.yaml/DEPS` listed in the target, rather than a `magic` logic handled in the backend logic.